readme.md

```diff
@@ -107,6 +107,18 @@ Settings available (in your VSCode workspace) for each of the linters:
 }
 ```
 
+## Completion
+
+You will need to install the ruby gem for Intellisense/Completion.
+
+* rcodetools
+
+You need to restart vscode after installation of rcodetools. Then type CTRL-Space after the leading words like below.
```

Comment:
To make this read better in English, maybe try this?

```
## Intellisense/Completion

To have completion in ruby, you will need to install the gem `rcodetools` via `gem install rcodetools`. For the first time, you will need to restart VS Code. For an example of it in action, try writing:

```ruby
[1, 2, 3].e #<= Press CTRL-Space here
```